wpilibc/athena/include/PeriodicRobot.h

```diff
@@ -0,0 +1,41 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) FIRST 2017. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <memory>
+
+#include "IterativeRobotBase.h"
+#include "Notifier.h"
+
+namespace frc {
+
+/**
+ * PeriodicRobot implements the IterativeRobotBase robot program framework.
+ *
+ * The PeriodicRobot class is intended to be subclassed by a user creating a
+ * robot program.
+ *
+ * Periodic() functions from the base class are called on an interval by a
+ * Notifier instance.
+ */
+class PeriodicRobot : public IterativeRobotBase {
+ public:
+  static constexpr double kDefaultPeriod = 0.01;
```

Comment:
I'd recommend a default of 0.02, to nominally match the driver station timing. That way there's minimal disruption to user code that depends on loop timing.